### PR TITLE
Move core into single package

### PR DESCRIPTION
**Pull request by frank-weinberg**
In the process simplfy structure from Interfaces view.X, model.XModel
and implementation DefaultXModel to single Interface core.X and
implementation core.implementation.XImpl

### CODE REVIEW

File: src/com/carolinarollergirls/scoreboard/core/ScoreBoard.java [hunk 1]
```diff
@@ -14,12 +14,16 @@
 import com.carolinarollergirls.scoreboard.xml.XmlScoreBoard;
 
 public interface ScoreBoard extends ScoreBoardEventProvider {
+    /** Reset the entire ScoreBoard. */
```
Comment (brian-brazil) [inline src/com/carolinarollergirls/scoreboard/core/ScoreBoard.java]: Remove "entire", as we don't reset Settings anymore.

File: src/com/carolinarollergirls/scoreboard/core/implementation/ClockImpl.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package com.carolinarollergirls.scoreboard.defaults;
+package com.carolinarollergirls.scoreboard.core.implementation;
```
Comment (brian-brazil) [inline src/com/carolinarollergirls/scoreboard/core/implementation/ClockImpl.java]: I think just impl would be enough, and it's faster to type.

File: src/com/carolinarollergirls/scoreboard/jetty/JettyServletScoreBoardController.java [hunk 3]
```diff
@@ -29,14 +29,13 @@
...
+import com.carolinarollergirls.scoreboard.core.ScoreBoard;
 import com.carolinarollergirls.scoreboard.json.JSONStateManager;
-import com.carolinarollergirls.scoreboard.model.ScoreBoardModel;
-// import org.eclipse.jetty.util.resource.Resource;
 
 
 public class JettyServletScoreBoardController {
-    public JettyServletScoreBoardController(ScoreBoardModel model, JSONStateManager jsm) {
-        scoreBoardModel = model;
+    public JettyServletScoreBoardController(ScoreBoard model, JSONStateManager jsm) {
```
Comment (brian-brazil) [inline src/com/carolinarollergirls/scoreboard/jetty/JettyServletScoreBoardController.java]: We should probably change args like this to `sb`

**Review by brian-brazil — COMMENTED** [review]
model is still used in names here and there.

File: src/com/carolinarollergirls/scoreboard/jetty/JettyServletScoreBoardController.java (hunk 3 above)
Comment (frank-weinberg) [reply src/com/carolinarollergirls/scoreboard/jetty/JettyServletScoreBoardController.java]: I've eliminated all occurences of the string 'model' from the java code now.

**Pull request merged**